npc/custom/jobmaster.txt

```diff
@@ -292,15 +436,146 @@ Get_Platinum:
 		break;
 	}
 	return;
+}
+
+// Function which gives a job related item to the player
+// the items are the rewards from the original job change quests
+function	Get_Job_Equip	{
+	// TODO checkweight, currently the item drops on the ground if the player can't hold it.
```

Comment:
I'll leave that as it is. It's better to drop the item than to not give it at all.